Get-NetView.psm1

```diff
@@ -115,6 +107,11 @@ $ExecFunctions = {
         Write-Output $commandOut
 
         Write-CmdLog "$logMsg"
+
+        if ($Scenario -eq "AzureHost") {
+            # Delay execution to avoid sustained CPU spikes.
+            Start-Sleep -Milliseconds 1000
```

Comment:
Don't hardcode, please make this a configurable value.  At 1sec the execution time is way too long...

The scenario setting would just use a proper API to set the delay value.  Azure scenario triggers this, then consider 100ms at the default.

Reply:
I don't really like the idea of making it configurable.
* Useful range of values is super narrow (~50 to 200 ms).
* It isn't clear what it should be set to. Paradoxically, it should be set to a *lower* value for higher-end systems. (Think about an extremely low-end system that takes 10+ seconds to execute each command. 100 ms of sleep means ~99% of the time is spent executing commands.)
* Net-Net there are more opportunities for end-users to sabotage themselves then improve functionality.

I think it would be better to sleep dynamically based on command duration? E.g. sleep for `n + x * duration` ms. High quality defaults are always better than configurability.

Reply:
- The delay to 1sec (1000ms), which goes against your first point.
- How long does it take to complete execution on a highly loaded VM or Container system?   
- The end user problem is addressed with Smart defaults applicable to the most common scenarios.
- IMHO, static values are simpler/safer here.  Else you will be diagnosing non-deterministic behavior across many different and conflicting environments.

If you consider the breadth of environments to execute, then it will require a tunable value.  In my examples I provided two options:

- -ExecDelay #defaults to 100ms and most applicable for the common Server target
- -ExecDelay <time-in-ms> #where users or wrapper can input the delay time.

Reply:
IMO dynamic sleep is the only viable option, because it automatically adjusts for available resources on live systems. e.g. Imagine collection starts on a system starts with 0% CPU use, but halfway thru, a customer starts a CPU intensive workload. In such a case, the delay needs to be increased to accommodate the customer workload.